src/log/log.c

```diff
@@ -2387,6 +2400,17 @@ __wt_log_scan(WT_SESSION_IMPL *session, WT_LSN *lsnp, uint32_t flags,
             if (LF_ISSET(WT_LOGSCAN_ONE))
                 break;
         }
+
+        /*
+         * Break the scanning loop if the next file is larger than the user set end range file or if
```

Comment:
I would word this as, "Exit the scanning loop if the next LSN seen is greater than our user set end range LSN."